AndelaOSP/andela-societies-frontend
# #[156914066] Feature to enable redeeming of society points

#### What does this PR do?
Enables redeeming of society points using the Redeem Points modal.
#### Description of Task to be completed?
Have the following working: 
 - Actions and action creators for redeeming points request, success, and failure.
 - Have a redeem points reducer to handle the actions.
 - Have a form to enable a user to redeem points.
 - A component to display form field errors.
 - A helper function to covert points entered to dollars.
 - Have tests for the redeemPointsReducer, redeemPointsActions, and RedeemPointsForm component.
#### How should this be manually tested?
 `git pull` this branch, `cd` into the project, `yarn install` the dependencies, and `yarn start`. Log into the application and navigate to `Redemptions` page. Click the floating button on the bottom-right corner to have the `Redeem points` modal appear.
#### What are the relevant pivotal tracker stories?
[156914066](https://www.pivotaltracker.com/story/show/156914066)
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-06-06 at 00 15 42" src="https://user-images.githubusercontent.com/29278184/41003253-f7c3a0c8-691e-11e8-8955-52b02e30eaf9.png">
<img width="1440" alt="screen shot 2018-06-06 at 00 15 49" src="https://user-images.githubusercontent.com/29278184/41003274-04f5ab56-691f-11e8-8ac9-60edbca6b026.png">
<img width="1440" alt="screen shot 2018-06-06 at 00 16 11" src="https://user-images.githubusercontent.com/29278184/41003283-08faae2c-691f-11e8-8cb1-3a6e3f519bee.png">
<img width="1440" alt="screen shot 2018-06-06 at 00 16 14" src="https://user-images.githubusercontent.com/29278184/41003284-0aa4223a-691f-11e8-9b0e-2d2455eff021.png">
